easybuild/easyconfigs/h/HPL/HPL-2.1-CrayGNU-5.1.29.eb

```diff
@@ -0,0 +1,18 @@
+name = 'HPL'
+version = '2.1'
+
+homepage = 'http://www.netlib.org/benchmark/hpl/'
+description = """HPL is a software package that solves a (random) dense linear system in double precision (64 bits) arithmetic 
+ on distributed-memory computers. It can thus be regarded as a portable as well as freely available implementation of the 
+ High Performance Computing Linpack Benchmark."""
+
+toolchain = {'name': 'CrayGNU', 'version': '5.1.29'}
+toolchainopts = {'optarch': True, 'usempi': True}
```

Comment:
I should drop `usempi` here, since it's basically a no-op when using the Cray compiler drivers...